packages/wallet/src/contract-tests/adjudicator-events.test.ts

```diff
@@ -41,7 +41,7 @@ describe("adjudicator listener", () => {
     await depositIntoETHAssetHolder(signer, channelId);
   });
 
-  it("should not handle an event when no process has registered", async () => {
+  it.skip("should not handle an event when no process has registered", async () => {
```

Comment:
Can these be re-enabled?

Reply:
I already spent some time trying to get these wallet tests to pass. Since we're going to delete the package, it doesn't make sense to continue with that.

Reply:
Sorry, I thought these were test in the `simple-hub` package not `wallet`